src/applications/disability-benefits/all-claims/components/Authorization.jsx

```diff
@@ -0,0 +1,500 @@
+              friends, public officials)
+            </li>
+          </ul>
+          <p>
+            You’ll have the option on the next page to limit your authorization
+            of types of sources and/or types of information.
+          </p>
+        </va-accordion-item>
+        <va-accordion-item header="3. Costs for records" level="4" open>
+          <p className="vads-u-margin-top--0">
+            VA won’t pay any fees charged by a custodian (source) to provide
+            records requested.
+          </p>
+        </va-accordion-item>
+        <va-accordion-item header="4. Penalties" level="4" open>
+          <p className="vads-u-margin-top--0">
+            The law provides severe penalties which include fine or
+            imprisonment, or both, for the willful submission of any statement
+            or evidence of material fact knowing it to be false.
+          </p>
+        </va-accordion-item>
+        <va-accordion-item
+          header="5. Validity of electronic records and signatures"
+          level="4"
+          open
+        >
+          <p className="vads-u-margin-top--0">
+            Under the Government Paperwork Elimination Act (GPEA) (Public Law
+            105-277), the Office of Management and Budget (OMB) ensures that
+            agencies, when practicable, provide for the option of electronic
+            maintenance, submission of disclosure of information and for the use
+            and acceptance of electronic signatures. GPEA states that electronic
+            records submitted or maintained in accordance with the procedures
+            developed by OMB, or electronic signature or other forms of
+            electronic authentication used in accordance with such procedures,
+            "shall not be denied legal effect, validity, or enforceability
+            merely because such records are in electronic form" (Public Law
+            105-277, section 1707).
+          </p>
+        </va-accordion-item>
+        <va-accordion-item
+          header="6. Submitting evidence and other mail"
+          level="4"
+          open
+        >
+          <p className="vads-u-margin-top--0">
+            You’ll have the option to upload documents later in this form. Or,
+            you can upload them later on{' '}
+            <va-link href="https://www.va.gov" text="VA.gov" />.
+          </p>
+          <p>
+            Documents may be submitted by mail, in person at a VA regional
+            office, or electronically. However, VA recommends submitting
+            correspondence electronically as this is the fastest method of
+            receipt.
+          </p>
+          <h5>Send by mail</h5>
+          <p className="va-address-block vads-u-margin-top--3">
+            Department of Veterans Affairs
+            <br />
+            Evidence Intake Center
+            <br />
+            PO Box 4444
+            <br />
+            Janesville, WI 53547-4444
+          </p>
```

Comment:
Consider extracting this address block, which appears in multiple places, into a shared component.

Reply:
👍 I will add this to the TODO list for the next PR.